crates/hir/src/lib.rs

```diff
@@ -4239,6 +4239,10 @@ impl Type {
         }
     }
 
+    pub fn fingerprint_for_trait_impl(&self) -> Option<TyFingerprint> {
```

Comment:
I guess for the changes we need to re-export `TyFingerprint` still? If we do that should go in the big re-export use statement

Reply:
weirdly enough, we don't need to reexport `TyFingerprint` for this method. not sure why!